integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/distrox/DistroXEncryptedVolumeTest.java

```diff
@@ -114,7 +114,7 @@ protected void setupTest(TestContext testContext) {
     }
 
     @AfterMethod(onlyForGroups = { "azure_singlerg" })
-    public void tearDown(Object[] data) {
+    public void singleResourceGroupTearDown(Object[] data) {
```

Comment:
same here: in this case cleanup will triggered twice, since parent class also calling this

Reply:
so cleanupTestContext call should be removed from this method